lib/matplotlib/text.py

```diff
@@ -114,6 +114,7 @@ def _get_textbox(text, renderer):
     "fontvariant": ["variant"],
     "verticalalignment": ["va"],
     "fontweight": ["weight"],
+    "math_fontfamily": ["math_font"],
```

Comment:
```suggestion
```
Let's not add an alias for now. Redundancy should only be added if there is a large benefit. The above aliases fall in two categories: a) stripping a *font*-prefix; b) 1-2 char abbreviations that are used for some common parameter names throughout the library. Neither of which applies here. `math_font` is not significantly better than `math_fontfamily`.